### PR TITLE
PAAS-1737 more console logs and always put back the inital conf if jexperience was active

### PR DESCRIPTION
**Pull request by laurentfufu**
JIRA issue: https://jira.jahia.org/browse/PAAS-1737

Short description:

### CODE REVIEW

File: packages/common/restore.yml [hunk 1]
```diff
@@ -281,46 +267,44 @@ actions:
...
+    - log: "jExperience was active before the restore: ${globals.jexperience_active}"
+    - if (${globals.jexperience_active}):
+        - log: "Initial jExperience conf need to restored"
         - cmd[proc, cp]: |-
             mv /root/org.jahia.modules.jexperience.settings-global.cfg /data/digital-factory-data/karaf/etc/org.jahia.modules.jexperience.settings-global.cfg
           user: root
 
     # if the restored jexperience isn't the same as the intiale one,
     # then we install the initial
+    - log: "jExperience had to be reinstall: ${this.reinstall_jexperience}"
```
Comment (kevinvandel) [inline packages/common/restore.yml:280]: *reinstalled

File: packages/common/restore.yml [hunk 2]
```diff
@@ -281,46 +267,44 @@ actions:
...
         - cmd[proc]: |-
             echo "Upgrade jexperience module to initial v${globals.jexperience_active_version} now" >> ${globals.logsPath}
+        - log: "install jExperience v${globals.jexperience_active_version}"
         - upgradeJahiaModule:
             module: jExperience
             version: ${globals.jexperience_active_version}
 
     # if the env intialy don't have jexperience active,
     # then we clean the conf and nodegroups data
+    - log: "Restored jExperience had to be remove: ${this.remove_restored_jexperience}"
```
Comment (kevinvandel) [inline packages/common/restore.yml:291]: *removed

**Review by kevinvandel — CHANGES_REQUESTED** [review]

File: packages/common/restore.yml (hunk 1 above)
Comment (laurentfufu) [reply packages/common/restore.yml]: fixed

File: packages/common/restore.yml (hunk 2 above)
Comment (laurentfufu) [reply packages/common/restore.yml]: fixed

**Review by kevinvandel — APPROVED** [review]

**Pull request merged**